scripts/jenkins/jobs-ibs/cloud-mkcloud6-scenario-dvr.yaml

```diff
@@ -0,0 +1,25 @@
+- job:
+    name: 'cloud-mkcloud6-scenario-dvr'
+    node: cloud-trigger
+
+    triggers:
+      - timed: 'H 19 */2 * *'
```

Comment:
Oh, didn't know we had this. I would indeed prefer to have scenario run on a daily basis if we can afford to.